From: John Johansen <john.johansen@canonical.com>
Date: Fri, 3 Apr 2026 04:09:05 -0700
Subject: [PATCH 13/13] parser: fix FTBFS due to missing merge edit

When 5ae09cb98 parser: fix label match for default label values
was backported one of the edits was missed from being added to the
commit, resultng in local builds but failed upstream builds.

Add the missing edit

Fixes: 5ae09cb98 ("parser: fix label match for default label values")
Signed-off-by: John Johansen <john.johansen@canonical.com>
---
 parser/network.cc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/parser/network.cc b/parser/network.cc
index 5a95fcf..6ff518c 100644
--- a/parser/network.cc
+++ b/parser/network.cc
@@ -838,7 +838,7 @@ bool network_rule::gen_ip_conds(Profile &prof, std::list<std::ostringstream> &st
 				 * the null transition */
 				if (!prof.policy.rules->add_rule(buf.c_str(), priority,
 								 rule_mode, map_perms(perms),
-								 dedup_perms_rule_t::audit == audit_t::FORCE ? map_perms(perms) : 0,
+								 dedup_perms_rule_t::audit == AUDIT_FORCE ? map_perms(perms) : 0,
 								 parseopts))
 					return false;
 			}
